jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitProjects.kt

```diff
@@ -91,8 +91,7 @@ class SamDynamoDBCookieCutter : SamPythonProjectTemplate() {
 }
 
 abstract class SamNodeJsProjectTemplate : SamProjectTemplate() {
-    // TODO support NodeJs 10.10 at some point
-    override fun supportedRuntimes(): Set<Runtime> = setOf(Runtime.NODEJS8_10)
+    override fun supportedRuntimes(): Set<Runtime> = setOf(Runtime.NODEJS8_10, Runtime.NODEJS10_X)
```

Comment:
Hmmm should we just say NodeJsRuntimeGroup.runtimes()?

Reply:
No, I think we should be explicit here since we're also stating node8 and 10.x elsewhere

Reply:
k

Reply:
For a long term, we need to load these configs dynamically. The template author should provide what runtimes it supports.